index.html

```diff
@@ -11,9 +11,197 @@
+                        <td data-label="Phone">+1764156161</td>
+                        <td data-label="Email">chandersen@pixel.com </td>
+                    </tr>
+                    <tr>
+                        <td data-label="Navn">Christian Andersen</td>
+                        <td data-label="Company">Pixel Inc.</td>
+                        <td data-label="Level">Senior</td>
+                        <td data-label="Deals closed">315</td>
+                        <td data-label="State">California</td>
+                        <td data-label="Phone">+1764156161</td>
+                        <td data-label="Email">chandersen@pixel.com </td>
+                    </tr>
+                    <tr>
+                        <td data-label="Navn">Christian Andersen</td>
+                        <td data-label="Company">Pixel Inc.</td>
+                        <td data-label="Level">Senior</td>
+                        <td data-label="Deals closed">315</td>
+                        <td data-label="State">California</td>
+                        <td data-label="Phone">+1764156161</td>
+                        <td data-label="Email">chandersen@pixel.com </td>
+                    </tr>
+                    <tr>
+                        <td data-label="Navn">Christian Andersen</td>
+                        <td data-label="Company">Pixel Inc.</td>
+                        <td data-label="Level">Senior</td>
+                        <td data-label="Deals closed">315</td>
+                        <td data-label="State">California</td>
+                        <td data-label="Phone">+1764156161</td>
+                        <td data-label="Email">chandersen@pixel.com </td>
+                    </tr>
+                    <tr>
+                        <td data-label="Navn">Christian Andersen</td>
+                        <td data-label="Company">Pixel Inc.</td>
+                        <td data-label="Level">Senior</td>
+                        <td data-label="Deals closed">315</td>
+                        <td data-label="State">California</td>
+                        <td data-label="Phone">+1764156161</td>
+                        <td data-label="Email">chandersen@pixel.com </td>
+                    </tr>
+                    <tr>
+                        <td data-label="Navn">Christian Andersen</td>
+                        <td data-label="Company">Pixel Inc.</td>
+                        <td data-label="Level">Senior</td>
+                        <td data-label="Deals closed">315</td>
+                        <td data-label="State">California</td>
+                        <td data-label="Phone">+1764156161</td>
+                        <td data-label="Email">chandersen@pixel.com </td>
+                    </tr>
+                </tbody>
+                <tfoot>
+                    <tr>
+                        <td colspan="4">Total agents: 28 </td>
+                        <td colspan="2" class="footer-td">
```

Comment:
Class "footer-td" applied to some cells only and no styles applied for this class.